index.js

```diff
@@ -11,16 +11,19 @@ const crypto = require("crypto");
 app.use(express.json());
 
 // Enable CORS (https://developers.asana.com/docs/security)
-app.use(cors({
-  origin: "https://app.asana.com"
-}));
+app.use(
+  cors({
+    origin: "https://app.asana.com",
+  })
+);
 
 // Run before every API request
 app.use((req, res, next) => {
   // Assess timeliness (https://developers.asana.com/docs/timeliness)
   const expirationDate = req.query.expires_at || req.body.expires_at;
+  const newDate = new Date();
 
-  if (new Date().getTime() > expirationDate) {
+  if (newDate.getTime() > new Date(expirationDate).getTime()) {
```

Comment:
I think you meant to add a space between `newDate`

Reply:
`newDate` is a actually variable, but good callout -- I should actually rename that to `currentDate`